src/test/mochitest/browser_dbg-sourcemaps2.js

```diff
@@ -24,7 +24,7 @@ add_task(function*() {
   yield addBreakpoint(dbg, mainSrc, 4);
   is(getBreakpoints(getState()).size, 1, "One breakpoint exists");
   ok(
-    getBreakpoint(getState(), { sourceId: mainSrc.id, line: 4 }),
+    getBreakpoint(getState(), { sourceId: mainSrc.id, line: 4, column: 2 }),
```

Comment:
this might be a regression - before we were making sure that we didn't turn line breakpoints into column breakpoints...